### ISSUE TITLE
New Task: Roll-up similar DaVIT functions into ATN data portal

### ISSUE THREAD

**Issue by laurabrenskelle** [ioos/marine_life_data_network#79]
### What is being requested?

The meaning of this issue is unclear. More information is needed.

### Provide a description or any other important information.

This is from a Marine Life Data Network Trello card.

**Comment by mmckinzie** [comment]
Also dependent on issue #51

**Comment by MathewBiddle** [comment]
Closing as this is a visualization effort which we are not prioritizing at the moment.
